frameworks/Java/wizzardo-http/wizzardo-http-postgres.dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM techempower/wizzardo-http-base:0.1
```

Comment:
You'll have to replace this line with the full contents of `wizzardo-http-base.dockerfile`.  We no longer support these relative path base dockerfiles.

The ability to find and extend these framework-specific base files was something we had to build into the TFB toolset ourselves.  As far as we know, it's not supported by docker out of the box.  We became uncomfortable with the idea that our dockerfiles (at least the ones that used this feature) were not "real" dockerfiles that could be trivially built with a single `docker build` command -- it was almost as if we introduced a new keyword or syntax.

Without these framework-specific base files we have a lot more copy-and-pasted code (between test variants in each framework), but we think that is worth the upside of having vanilla dockerfiles.  Fortunately docker's built-in caching is able to see when builds share copy-and-pasted steps and it doesn't need to repeat them, and our toolset now takes advantage of that caching (by not deleting build artifacts between tests like it used to).